package.json

```diff
@@ -45,7 +46,6 @@
     "browserify": "^12.0.1",
     "eslint": "^1.10.3",
     "eslint-plugin-react": "^3.11.3",
-    "jasmine": "^2.4.1",
```

Comment:
The library has a name "jasmine-expect-jsx". It should work for both jasmine & jest.